packages/utilities/src/useFocusRects.ts

```diff
@@ -40,9 +40,9 @@ type AppWindow = (Window & { FabricConfig?: { disableFocusRects?: boolean } }) |
  * @param rootRef - A Ref object. Focus rectangle can be applied on itself and all its children.
  */
 export function useFocusRects(rootRef?: React.RefObject<HTMLElement>): void {
```

Comment:
This is still not enough safe it will be never reexecuted if `rootRef.current` will change